cmd/grype-db/cli/options/package.go

```diff
@@ -12,9 +12,6 @@ type Package struct {
 	PublishBaseURL           string            `yaml:"publish-base-url" json:"publish-base-url" mapstructure:"publish-base-url"`
 	OverrideArchiveExtension string            `yaml:"override-archive-extension" json:"override-archive-extension" mapstructure:"override-archive-extension"`
 	CompressorCommands       map[string]string `yaml:"compressor-commands" json:"compressor-commands" mapstructure:"compressor-commands"`
-
-	// unbound options
-	compressorCommandsJSON string
```

Comment:
Removing this field is not logically part of this change, but it was failing static analysis because it's not exported or used.